tests/test_event_handlers.py

```diff
@@ -99,6 +104,11 @@ def test_customer_created(self, event_retrieve_mock, customer_retrieve_mock):
     @patch("stripe.Customer.retrieve")
     @patch("stripe.Event.retrieve")
     def test_customer_created_no_customer_exists(self, event_retrieve_mock, customer_retrieve_mock):
+        """Test customer_webhook_handler.
```

Comment:
Sorry, but please remove all docstrings on tests. When tests fail (or verbosity is on), the dosctrings are shown instead of the method names which makes finding the failing test more difficult than it should be